src/libraries/System.Drawing.Common/src/System.Drawing.Common.csproj

```diff
@@ -8,8 +8,12 @@
     <TargetFrameworks>$(NetCoreAppCurrent)-Windows_NT;$(NetCoreAppCurrent)-Unix;$(NetCoreAppCurrent)-Browser;netcoreapp3.0-Windows_NT;netcoreapp3.0-Unix</TargetFrameworks>
     <ExcludeCurrentNetCoreAppFromPackage>true</ExcludeCurrentNetCoreAppFromPackage>
     <Nullable>enable</Nullable>
+    <IgnoreForCI Condition="'$(TargetOS)' == 'Browser'">true</IgnoreForCI>
```

Comment:
Why does this use TargetOS == Browser and below it uses TargetsBrowser?

Reply:
`TargetsBrowser` will only be set if the project is cross compiled for `Browser` which in this case it is. So I guess we can just use `TargetsBrowser` here. You want to use `TargetOS == Browser` when it is not guaranteed that a project is cross compiled, i.e: in a common location like Directory.Build.targets.

Reply:
Actually this has to be part of the test project, not the source project.

Reply:
Fixed